package.json

```diff
@@ -42,6 +46,7 @@
     "sharp-aws-image-handler-client": "https://github.com/stelace/sharp-aws-image-handler-client.git#0.2.0",
     "socket.io-client": "~2.3.0",
     "stelace": "0.13.0",
+    "stripe": "^8.14.0",
```

Comment:
Is `stripe` dependency used outside of netlify functions?

Reply:
No indeed, I missed to move it to Netlify dependencies only.